modules/learn/pages/clusters-and-availability/failover.adoc

```diff
@@ -150,6 +150,24 @@ Ratios on nodes 1 to 3 are now _1:2_, indicating that rebalance has reduced the
 
 For further examples of rebalance in the context of _node removal_, see xref:learn:clusters-and-availability/removal.adoc[Removal].
 
+[#failover-efficiency]
+== Failover Efficiency
+
+The efficiency with which failover occurs is related to the cluster's xref:learn:clusters-and-availability/cluster-manager.adoc#master-services[Master Services].
+These manage operations with cluster-wide impact; such as failover, rebalance, and adding and deleting buckets.
+The _Master Services_ are sometimes referred to as the _Orchestrator_.
+
+At any given time, only one instance of the Master Services is in charge: the instances having negotiated among themselves, to identify and elect the instance.
+Should the elected instance subsequently become unavailable, another takes over.
+
+When the node that is currently hosting the active Master Services must itself be failed over, this requires greater latency than usual; since time must be taken to ensure that negotiation and election of a replacement occur.
+In case where the Master Services are co-located with a highly subscribed services (in particular, the Data Service), this can result in unnecessary latency in failing over the node and ensuring that data can be served.
+
+In consequence, the Master Services should ideally _not_ be co-located with any service.
+Such a configuration is possible in Couchbase Enterprise Server 7.6+: one or more _serviceless_ nodes can be added to the cluster, with the intention of ensuring that the Master Services will occupy such a node, and thereby not be co-located with any service.
```

Comment:
@stevewatanabe please correct me if I'm wrong, but there's no requirement to use an arbiter node to achieve this goal.   The cluster will automatically choose any node that isn't running the data service as a preference.   The addition of an arbiter node is useful where there are no available nodes in the cluster in the required configuration such as a cluster of all data nodes.

Reply:
I've asked @BenHuddleston to chime in as he implemented a weighting system based on the services and their type running on each node. Ian is correct that an arbiter node isn't required....Ben can provide more detailed information.

Reply:
@b33f  any non-KV node could be used for this. For your interest, the priority order is as follows (higher weight => lower priority, service-less nodes are implicitly 0):

```
%% Weight values used in mb_master to determine orchestrator placement.
-define(DEFAULT_SERVICE_WEIGHTS, [{kv, 10000},
                                  {index, 1000},
                                  {fts, 1000},
                                  {cbas, 1000},
                                  {n1ql, 100},
                                  {eventing, 100},
                                  {backup, 10}]).
```